### ISSUE TITLE
[Discussion] Media Management for FPP

### ISSUE THREAD

**Issue by kylegrymonprez** [xLightsSequencer/xLights#4854]
**Is your feature request related to a problem? Please describe.**
One of the common things people are doing is uploading a video file to remote FPP instances in order to 'trick' HDMI out using the full video vs sequence data on that remote pi.

**Describe the solution you'd like**
Should we add some sort of multiple media management abilities to xLights and FPP Connect in order to manage which media files should do to which FPP devices? I think it could be interesting.

**Comment by cybercop23** [comment]
That's already there. Create a folder with the same exact FPP name in your media folder and put the MP4 with the same exact name as the sequence mp3 and check media upload in FPP connect. It will first go through this process to find act matching items and load them to the appropriate FPP.
Attentively you can also rename the media/MP4 and add the same exact FPP name to the end of the filename and will have the same effect as above. But I think having a separate folder per FPP instance if better file management.

**Comment by derwin12** [comment]
Can we make this easier? Perhaps a placeholder effect in the sequence? Just tossing out ideas..

**Comment by cybercop23** [comment]
They have to manage and create the content, assuming it's not just the sequence media, if it is, checking that box will just upload it. You don't want FPP connect to search through the sequence file to find it and then there's multiple VMs or even split content among left/right projector.

**Comment by kylegrymonprez** [comment]
yeah - i'm thinking more like file X goes to FPP instance A, file B goes to FPP instance B and FPP connect takes all of the file naming and stuff into account on it's own. Just a mapping of file on compter to FPP instance, and the upload process handles the file naming stuff.

**Comment by cybercop23** [comment]
and why not use Discussions... :)

**Comment by kylegrymonprez** [comment]
(oops forgot about discussions - I think someone can move it over - I can't)
